src/magic_folder/config.py

```diff
@@ -1369,3 +1357,25 @@ def create_magic_folder(self, name, magic_path, state_path, author,
                 )
 
         return mfc
+
+
+def _validate_listen_endpoint_str(ep_string):
+    """
+    confirm we have a valid endpoint-string
+    """
+    from twisted.internet import reactor  # uhm...
+    # XXX so, having the reactor here sucks. But if we pass in an
+    # IStreamServerEndpoint instead, how can we turn that back
+    # into an endpoint-string?
```

Comment:
not sure this comment still makes sense in this context, in particular about turning it back into a string, since this function is just concerned with validating the string.